apps/pie-docs/src/patterns/cookie-banner/code/code.md

```diff
@@ -0,0 +1,161 @@
+---
+eleventyNavigation:
+    key: Code
+    parent: 'Cookie Banner'
+    order: 3
+shouldShowContents: true
+eleventyComputed:
+    props: "{% include './props.json' %}"
+    events: "{% include './events.json' %}"
+---
+
+## Overview
+
+<p>
+  <a href="https://www.npmjs.com/@justeattakeaway/pie-cookie-banner">
+    <img alt="GitHub Workflow Status" src="https://img.shields.io/npm/v/@justeattakeaway/pie-cookie-banner.svg">
+  </a>
+</p>
+
+`pie-cookie-banner` is a Web Component built using the Lit library.
+
+This component can be easily integrated into various frontend frameworks and customized through a set of properties.
+
+## Cookies
+`pie-cookie-banner` purely handles the UI and user interactions for cookie consent. It does not handle the setting or management of cookies or the storage of user preferences. It is the responsibility of the consuming application to handle these aspects. See [Events](http://localhost:8080/patterns/cookie-banner/code/#events) for more information.
```

Comment:
This link is currently pointing at localhost 😄

Reply:
🙈